index.js

```diff
@@ -75,6 +75,18 @@ const publicApi = {
         return this;
     },
 
+    setFontsPublicPath(publicPath) {
+        webpackConfig.setFontsPublicPath(publicPath);
+
+        return this;
+    },
+
+    setImagesPublicPath(publicPath) {
```

Comment:
Missing comment for that one too.

Reply:
By the way, do we really need two methods? is there a use case in which we would only call one of them or call them with different values?